new/AT1147CaliphAtrib.xml

```diff
@@ -0,0 +1,67 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="AT1147CaliphAtrib" xml:lang="en" type="auth">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>A Caliph from ʾAtrib tries to destroy a Church</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="GS"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine
+                    multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>A miniature with four episodes, showing: first, the Caliph ordering his troops to
+                    destroy a Church in ʾAtrib; second, <persName ref="PRS6819Mary"/> appearing to the
+                    Caliph (who was asleep) to order his agent to stop the destruction of the
+                    Church with a white dove appearing to take his letter between them; third, the
+                    dove giving the letter written by the caliph to the head of troops (agent of
+                    the Caliph) with the Abbot of the Church standing in the gate, and
+                    fourth, the Caliph who later become believer sitting in front of the Church.</p>
+
+            </abstract>
+            <textClass>
+
+                <keywords>
+                    <term key="Apocrypha"/>
+                    <term key="Miracle"/>
+                    <term key="Hagiography"/>
+                    <term key="ChristianContent"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            <langUsage>
```

Comment:
always before submitting anything check in your oxygen that the document is valid (green). This document is not well formed XML
in line 55 you need the closing tag `</langUsage>`, not the opening tag